app/features-json/setting_json_controller.rb

```diff
@@ -0,0 +1,29 @@
+require_relative "api_controller"
+
+module FastlaneCI
+  # Controller for providing all data relating to system settings
+  class SettingJSONController < APIController
+    HOME = "/data/settings"
+
+    get HOME do
+      return json(Services.setting_service.settings)
+    end
+
+    post HOME do
+      metrics_setting = Services.setting_service.find_setting(setting_key: :metrics_enabled)
+      metrics_setting.value = false
```

Comment:
always false, let's pull this from the post body

Reply:
Hehe, yah, this is just code to try out setting a value using an HTTP request, this will be removed for sure 👍